xml/ha_config_cli.xml

```diff
@@ -1275,13 +1275,13 @@ hostname (string): Hostname
     -->
 <screen>&prompt.crm.conf;<command>location</command> loc-fs1 fs1 100: &node1;</screen>
     <para>
-     Another example is a location with pingd:
+     Another example is a location with ping:
     </para>
-<screen>&prompt.crm.conf;<command>primitive</command> pingd pingd \
-    params name=pingd dampen=5s multiplier=100 host_list="r1 r2"
+<screen>&prompt.crm.conf;<command>primitive</command> ping ping \
+    params name=ping dampen=5s multiplier=100 host_list="r1 r2"
```

Comment:
Probably we should point out the parameter `host_list` is a space separated list of hosts to ping and count?

I realized the primitive resource should be cloned so that each node has a running instance:

```
clone cl-ping ping \
    meta interleave=true
```

Reply:
> Probably we should point out the parameter `host_list` is a space separated list of hosts to ping and count?

Good point. I've added:

    The parameter <parameter>host_list</parameter> is a space-separated list
    of hosts to ping and count.

> I realized the primitive resource should be cloned so that each node has a running instance:
> 
> ```
> clone cl-ping ping \
>     meta interleave=true
> ```

Ok, I have now this:

```
crm(live)configure# primitive ping ping \
    params name=ping dampen=5s multiplier=100 host_list="r1 r2"
crm(live)configure# clone cl-ping ping meta interleave=true
crm(live)configure# location loc-node_pref internal_www \
    rule 50: #uname eq alice \
    rule ping: defined ping
```